src/NIDmmDCRmsMeasurement/NIDmmDCRmsMeasurement.lvproj

```diff
@@ -113,83 +115,6 @@
 				<Property Name="NI.PreserveRelativePath" Type="Bool">true</Property>
 			</Item>
 		</Item>
-		<Item Name="Build Specifications" Type="Build">
```

Comment:
Why did the `exe` build spec get deleted?

Reply:
We need the exe build spec kept in the project. We just don't need to upload the built exe itself uploaded to Github.